server/data/probationTeamsClient.js

```diff
@@ -21,6 +21,11 @@ module.exports = signInService => {
     async getFunctionalMailbox(probationAreaCode, lduCode, teamCode) {
       const ldu = await get(`${apiUrl}/probation-areas/${probationAreaCode}/local-delivery-units/${lduCode}`)
       const teamAddress = R.path(['probationTeams', teamCode, 'functionalMailbox'], ldu)
+      if (!(teamAddress || (ldu && ldu.functionalMailbox))) {
```

Comment:
This isn't very clear. Rather than duplicating complex booleans - it's better to extract and reuse..

So in this case what does `teamAddress || (ldu && ldu.functionalMailbox)` represent?

Reply:
changed to 

const functionalMailbox = teamAddress || (ldu && ldu.functionalMailbox) 
if (!functionalMailbox) {
    logger.info(`Could not find functional mailbox for the probation area code ${probationAreaCode} having an ldu code: ${lduCode} or team code: ${teamCode}` )
}
return functionalMailbox